View Controllers/ProfileViewController.m

```diff
@@ -69,6 +69,12 @@ - (void)viewDidLoad {
     
     // create map
     GMSCameraPosition* camera = [GMSCameraPosition cameraWithLatitude:37.7749 longitude:-122.4194 zoom:6];
+    
+    //make sure we have the right width
```

Comment:
I might elaborate on this comment a little bit about why profileMapView's bounds are wrong in the first place, as this is a bit of a hack